api/testing-internal/src/main/java/io/opentelemetry/api/testing/internal/AbstractOpenTelemetryTest.java

```diff
@@ -88,16 +91,17 @@ void independentNonGlobalPropagators() {
   @Test
   void setThenSet() {
     setOpenTelemetry();
-    assertThatThrownBy(() -> GlobalOpenTelemetry.set(getOpenTelemetry()))
+    Assertions.assertThatThrownBy(() -> GlobalOpenTelemetry.set(getOpenTelemetry()))
```

Comment:
nit: why not use the static import that was here before?

Reply:
same question applies above in `AbstractDefaultTracerTest.java`

Reply:
Following the general convention in the project to import individual static methods for `Assertions` to reduce verbosity.

Reply:
But, you removed the static import...

Reply:
Oh I see.. Yeah that was unintentional artifact of moving stuff around in intellij. WIll fix.